tests/transactions/btc.fixtures.ts

```diff
@@ -0,0 +1,169 @@
+import BigNumber from 'bignumber.js';
+import { UTXO } from 'types';
+import { Recipient } from '../../transactions/btc';
+
+import { utxo10k, utxo384k, utxo3k, utxo792k } from './btc.data';
+
+type FixtureName = string;
+type FeeRate = number;
+
+type SelectUtxoSendSuccessExpected = {
+  selectedUtxos: UTXO[];
+  change: number;
+  fee: number;
+};
+
+type SelectUtxoSendSuccessFixture = [FixtureName, Recipient[], FeeRate, SelectUtxoSendSuccessExpected];
+
+export const recipientAddress1 = 'bc1pgkwmp9u9nel8c36a2t7jwkpq0hmlhmm8gm00kpdxdy864ew2l6zqw2l6vh';
+export const recipientAddress2 = 'bc1pyzfhlkq29sylwlv72ve52w8mn7hclefzhyay3dxh32r0322yx6uqajvr3y';
+
+export const selectUtxosForSendSuccessFixtures: SelectUtxoSendSuccessFixture[] = [
+  [
+    'Large output, prefer largest UTXO with largest change',
+    [
+      {
+        address: recipientAddress1,
+        amountSats: new BigNumber(60000),
+      },
+    ],
+    10,
+    {
+      selectedUtxos: [utxo792k],
+      change: 730120,
+      fee: 1880,
+    },
```

Comment:
nit: i'd use named parameters and destructuring for these

Reply:
That's a great idea 👍